### PR TITLE
Fix: Test on LTS and Stable

### LINKED ISSUE #617: Update CI scripts to use LTS and Stable versions of sonar
node has released version 9.

We should update our scripts to test in LTS and Stable to make sure we don't break anyone (or if we do, at least it's documented).

Ref: https://github.com/sonarwhal/online-service/pull/44#issuecomment-340907307

### PR DESCRIPTION
**Pull request by molant**
Fix #617

## Pull request checklist

Make sure you:

- [X] Signed the [Contributor License Agreement](https://cla.js.foundation/sonarwhal/sonar)
- [X] Followed the [commit message guidelines](https://sonarwhal.com/docs/contributor-guide/contributing/pull-requests/#commitmessages)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [x] Added/Updated related tests.

### CODE REVIEW

**Comment by molant** [comment]
@alrra do you know why we have 3 Linux tests and only 1 MacOS?

**Comment by molant** [comment]
Also we might have to update to [canvas-prebuilt 2.0.0-alpha5](https://github.com/node-gfx/node-canvas-prebuilt/issues/26) to make it work on node 9

**Comment by molant** [comment]
So https://github.com/node-gfx/node-canvas-prebuilt/issues/26 is finished and AppVeyor is already passing. Need to figure out what's wrong with the Travis configuration but this should be closed soon.

**Comment by molant** [comment]
@alrra with the current change it looks like we have the right build matrix (pending getting macOS builds to start and finish, issues with Travis again 😔).

Let me know if that looks OK to you and I'll squash and merge.

File: .travis.yml [hunk 1]
```diff
@@ -25,16 +21,29 @@ matrix:
...
 
+      node_js: lts/*
+      if: type = cron
       script:
         - ./.travis/check-links.sh
 
     # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 
-    # Build used for running the tests on Ubuntu.
+    # Build used for running the tests on Ubuntu on LTS
```
Comment (alrra) [inline .travis.yml]: > latest LTS.

File: .travis.yml [hunk 2]
```diff
@@ -25,16 +21,29 @@ matrix:
     # (is automatically run once per week).
 
     - os: linux
-      if: type = cron
```
Comment (alrra) [inline .travis.yml]: For consistency with the rest, remove this empty line.

File: .travis.yml [hunk 3]
```diff
@@ -44,10 +53,23 @@ matrix:
 
     # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 
-    # Build used for running the tests on macOS.
+    # Build used for running the tests on macOS on LTS/*.
```
Comment (alrra) [inline .travis.yml]: > latest LTS.

**Comment by alrra** [comment]
> Let me know if that looks OK 

I wanted somehow to avoid the repetition, but let's go with this for now.

**Comment by alrra** [comment]
Merged with some modifications.

**Pull request closed without merging**